src/Provider/TileProvider.js

```diff
@@ -73,15 +74,9 @@ function executeCommand(command) {
     }
 
     // build tile
-    const params = {
-        extent,
-        level,
-        materialOptions: layer.materialOptions,
-    };
-
     geometry._count++;
-    const tile = new TileMesh(geometry, params);
-    tile.layer = layer;
+    const material = new LayeredMaterial(layer.materialOptions);
```

Comment:
Could you add somewhere a documentation to what is `materialOptions` ? Even if it is a quick comment, so we do no forget when we'll generate a better documentation on layers.

Reply:
Ok. Note that these options are a workaround to limitations in the current code.

Hopefully these limitations will be removed soon and so will the options :)